charts/vh-admin-web/values.dev.template.yaml

```diff
@@ -8,4 +8,5 @@ java:
     AZUREAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
     AZUREAD__REDIRECTURI: https://${SERVICE_FQDN}/home
     DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-    DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+    DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+    
```

Comment:
reset this to master if there's no real change

Reply:
I used this pr to point at the user-api, have removed the template url which is why it's showing a change